tiemndo/pom.xml

```diff
@@ -48,11 +52,11 @@
             <artifactId>h2</artifactId>
 <!--            <scope>runtime</scope>-->
         </dependency>
-<!--        <dependency>-->
-<!--            <groupId>org.postgresql</groupId>-->
-<!--            <artifactId>postgresql</artifactId>-->
```

Comment:
We gotta remember to look at this before we deploy

Reply:
I had a change in plans, I'm going to temporarily deploy to my own heroku for now, but not push the changes that I'm making to do so, since we don't want our db passwords on github. We'll definitely have to figure out a better solution moving forward though